src/modules/powerrename/PowerRenameUILib/MainWindow.xaml

```diff
@@ -121,17 +121,33 @@
             </ListView>
         </Grid>
 
-        <Grid Grid.Column="0" Grid.Row="1" VerticalAlignment="Stretch" HorizontalAlignment="Stretch">
+        <Grid Grid.Column="0" Grid.Row="1" MaxWidth="360" VerticalAlignment="Stretch" HorizontalAlignment="Stretch">
             <Grid.RowDefinitions>
                 <RowDefinition Height="*" />
                 <RowDefinition Height="48" />
             </Grid.RowDefinitions>
-            <ScrollViewer>
+          
                 <StackPanel Orientation="Vertical" Padding="0,0,0,16" Margin="0,0,12,0">
                     <Grid>
-                        <AutoSuggestBox x:Name="textBox_search" x:Uid="SearchBox" Height="40" VerticalContentAlignment="Center" ItemsSource="{x:Bind SearchMRU}" />
-
-                        <Button FontFamily="{ThemeResource SymbolThemeFontFamily}" VerticalAlignment="Center" Visibility="{Binding ElementName=checkBox_regex, Path=IsChecked}" MinHeight="32" Margin="4" x:Uid="RegExButton" Background="Transparent" BorderBrush="Transparent" Content="&#xE946;" HorizontalAlignment="Right">
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="*"/>
+                        <ColumnDefinition Width="Auto"/>
+                    </Grid.ColumnDefinitions>
+                    <AutoSuggestBox x:Name="textBox_search" x:Uid="SearchBox" Height="40" VerticalContentAlignment="Center" ItemsSource="{x:Bind SearchMRU}" />
+
+                        <Button Grid.Column="1"
+                                FontFamily="{ThemeResource SymbolThemeFontFamily}"
+                                VerticalAlignment="Center"
+                                Visibility="{Binding ElementName=checkBox_regex, Path=IsChecked}"
+                                Height="40"
```

Comment:
Should the other button like this one also have Height set?

Reply:
Fixed